perf/twoPodTest/templates/fortio.yaml

```diff
@@ -22,6 +45,23 @@ spec:
     app: {{ $.name }}
   type: ClusterIP
 ---
+{{- if $.Values.rbac.enabled }}
+{{- range $i, $e := until ($.Values.rbac.numPolicies|int) }}
+{{- $data := dict "name" $.name "Values" $.Values "userName" (randAlpha 16) }}
+{{- template "rbac" $data }}
+---
+{{- end }}
+{{- $data := dict "name" .name "Values" .Values "ruleName" (randAlphaNum 16) "userName" "*" }}
```

Comment:
Sure, this is used to generate a RBAC rule with random name.
line 51 calls the `rbac` template function defined between line 1 and 22. It generates a RBAC rule with some variables, for example the `ruleName` is a random string assigned in line 2.

line 49 - 50 calls line 51 `$.Values.rbac.numPolicies` times so that we will have `$.Values.rbac.numPolicies` number of random RBAC rules.

line 54 is a special rule that generates a single RBAC rule with `userName` set to `*`, this allows anyone to access the service so that the perf test could be done.

Because each generated RBAC rule has a random name, so the order of the "allow all" rule will be also random in the Envoy RBAC filter which is good for the perf test.

If you're interested in the detail helm grammar and syntax, you can find it [here](https://docs.helm.sh/chart_template_guide/#the-chart-template-developer-s-guide)

Reply:
Sounds good. Thanks!